.gitignore

```diff
@@ -7,10 +7,7 @@ coverage/
 tsdocs/
 tsconfig.tsbuildinfo
 database.sqlite
-packages/**/*.d.ts
-packages/**/*.d.ts.map
-packages/**/*.js
-packages/**/*.js.map
+packages/**/build
```

Comment:
For my own understanding, why this change?

Reply:
It's just fewer things to keep in mind - everything we build goes into `build`, so we can just blanket ignore the folder as a whole, which allows us to have js files elsewhere if we ever need them.